client/rpc-api/src/system/mod.rs

```diff
@@ -108,4 +108,9 @@ pub trait SystemApi<Hash, Number> {
 	/// known block.
 	#[rpc(name = "system_syncState", returns = "SyncState<Number>")]
 	fn system_sync_state(&self) -> Receiver<SyncState<Number>>;
+
+	/// Reloads the logging filter, replacing current with the supplied directives
+	#[rpc(name = "system_reloadLogFilter", returns = "()")]
```

Comment:
```suggestion
	/// Sets the logging filter, replacing current with the supplied directives
	#[rpc(name = "system_setLogFilter", returns = "()")]
```

I don't really see why this says "reload". From the user's point of view we just change the filter.